tests/e2e/test_pvc_creation_performance.py

```diff
@@ -167,6 +165,15 @@ def test_multiple_pvc_creation_after_deletion_performance(
             number_of_pvc=number_of_pvcs,
             size=self.pvc_size,
         )
+        for pvc_obj in pvc_objs:
+            pvc_obj.reload()
+            teardown_factory(pvc_obj)
+        with ThreadPoolExecutor() as executor:
+            for pvc_obj in pvc_objs:
+                executor.submit(
```

Comment:
helpers.wait_for_resource_state will be started in a separate thread. The main thread will continue without actually waiting for the status.